sdk-http-ahc-2_5/src/main/java/io/sphere/sdk/http/AsyncHttpClientAdapter.java

```diff
@@ -0,0 +1,13 @@
+package io.sphere.sdk.http;
+
+import org.asynchttpclient.AsyncHttpClient;
+
+/**
+ * Adapter to use {@code AsyncHttpClient} (version 2.0.x) as {@code HttpClient}.
```

Comment:
```suggestion
 * Adapter to use {@code AsyncHttpClient} (version 2.4.x) as {@code HttpClient}.
```

Reply:
good that you pointed it out , i was referring to 2.5.4, i will fix the docs